packages/workspace/src/generators/remove/lib/check-targets.ts

```diff
@@ -43,3 +53,24 @@ export function checkTargets(
     throw new Error(message);
   }
 }
+
+function checkIfProjectIsUsed(
```

Comment:
Out of scope of this PR, but we should update this to check if it's a local plugin in use